diff --git a/vendor/gedmo/doctrine-extensions/src/Loggable/Entity/MappedSuperclass/AbstractLogEntry.php b/vendor/gedmo/doctrine-extensions/src/Loggable/Entity/MappedSuperclass/AbstractLogEntry.php
index d096554..730c068 100644
--- a/vendor/gedmo/doctrine-extensions/src/Loggable/Entity/MappedSuperclass/AbstractLogEntry.php
+++ b/vendor/gedmo/doctrine-extensions/src/Loggable/Entity/MappedSuperclass/AbstractLogEntry.php
@@ -83,11 +83,9 @@ abstract class AbstractLogEntry implements LogEntryInterface
     /**
      * @var array<string, mixed>|null
      *
-     * @ORM\Column(type="array", nullable=true)
-     *
-     * NOTE: The attribute uses the "array" name directly instead of the constant since it was removed in DBAL 4.0.
+     * @ORM\Column(type="json", nullable=true)
      */
-    #[ORM\Column(type: 'array', nullable: true)]
+    #[ORM\Column(type: Types::JSON, nullable: true)]
     protected $data;
 
     /**
